### DISCUSSION TITLE
Problem with HWinstall

### DISCUSSION THREAD

**Discussion by warnerthuis** [netalertx/NetAlertX#817] [General]
I have a proxmox with a clean Debian12 on it and did the installation via wget. At the end there was someting that looked like an information but I didn't copied it. When I restart the machine and on port 80 I get the Apache2 debian default page. on :20211 I get nothing, just a blank page. Does it start automatically or do I have to do someting to get that done?
I ran the install again and got the following error: (as I can't copy from the console I made a screenshot)
![error netalertx](https://github.com/user-attachments/assets/34aaaeec-2bb9-4d41-bc24-08973dbedba6)

**Comment by jokob-sk** [comment]
Hey @warnerthuis , this is what ChatGPT says:

# Troubleshooting `netifaces` Installation Error

The `legacy-install-failure` error during the installation of the `netifaces` package usually indicates a problem with building the package from source. Here are some steps to troubleshoot and resolve the issue:

## 1. Upgrade `pip`, `setuptools`, and `wheel`
Make sure you have the latest version of `pip`, `setuptools`, and `wheel`: `pip install --upgrade pip setuptools wheel`

## 2. Install Required Build Tools
On some systems, you may need to install additional build tools or libraries. Here’s how to do this for different operating systems:

- **Ubuntu/Debian:** `sudo apt update && sudo apt install python3-dev build-essential`
- **Fedora:** `sudo dnf install python3-devel gcc`
- **macOS:** `xcode-select --install`

## 3. Install `netifaces` from a Wheel File
If you're still facing issues, you can try to install `netifaces` using a precompiled wheel file: `pip install /path/to/netifaces.whl`. You can find wheel files at [PyPI](https://pypi.org/project/netifaces/#files).

## 4. Check Python Version and Environment
Make sure you are using a compatible version of Python for the `netifaces` package. Additionally, consider using a virtual environment: `python3 -m venv myenv && source myenv/bin/activate && pip install netifaces`.

## 5. Check Logs for Specific Errors
If the installation fails again, check the terminal output for any specific error messages that can give clues about what went wrong.

## 6. Consult the Documentation
If you're still unable to resolve the issue, consult the [netifaces documentation](https://pypi.org/project/netifaces/) or consider filing an issue on the [project's GitHub page](https://github.com/al45tair/netifaces).


Afterwards re-run the install
